cypress/support/fragments/settings/notes/noteTypes.js

```diff
@@ -22,4 +33,50 @@ export default {
       isDefaultSearchParamsRequired: false,
     });
   },
+
+  waitLoading: () => {
+    cy.expect(noteTypePane.exists());
+  },
+
+  addNoteType: () => {
+    cy.do(newNoteTypeButton.click());
+    cy.expect([
+      cancelNoteTypeCreationButton.exists(),
+      saveNoteTypeButton.has({ disabled: true }),
+      noteTypeInput.exists(),
+    ]);
+  },
+
+  fillInNoteType: (noteTypeName) => {
+    cy.do(noteTypeInput.fillIn(noteTypeName));
+    cy.expect(saveNoteTypeButton.has({ disabled: false }));
+  },
+
+  saveNoteType: (noteType) => {
+    cy.do(saveNoteTypeButton.click());
+    cy.wait(2000);
```

Comment:
Add comment here please about why you have used waiter here.

Reply:
done